app/javascript/memo.vue

```diff
@@ -27,7 +27,20 @@ export default {
     return {
       sendBody: '',
       id: '',
-      body: ''
+      body: '',
+      checked: false,
+    }
+  },
+  watch: {
+    body: function(newBody, oldBody) {
+      this.checked = false;
+    },
+    checked: function(newChecked, OldChecked) {
```

Comment:
```suggestion
    checked: function(newChecked, oldChecked) {
```

javascriptは慣習的に変数は小文字始まりですね。

あと使わない引数は`_`始まりにするとかもプロジェクトによってはルールがあります。その場合は`_oldChecked`です。

Reply:
指摘ありがとうございます！
単純にミスってました。
使わない変数は消しました！